internal/plugin_test.go

```diff
@@ -4,7 +4,7 @@ package internal
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
```

Comment:
why the change in imports?

Reply:
`ioutil` is deprecated: https://pkg.go.dev/io/ioutil

Reply:
gotcha, thanks!